src/main/scala/org/holmesprocessing/totem/services/pemeta/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM golang:alpine
+
+# Create folder
+RUN mkdir -p /service
+WORKDIR /service
+
+# Get Golang and PEV dependencies
```

Comment:
please follow the format for the other docker files. Following the same layers makes the build process faster due to Docker caching. 

Take a look at virustotal's service for an example.